app/styles/nypr-ui/_story.scss

```diff
@@ -0,0 +1,55 @@
+article figure {
+  margin-bottom: 30px;
+}
+
+article footer {
+  max-width: 630px;
```

Comment:
No, we're making the body of the article narrower to improve readability. Been meaning to do that for a while.